### ISSUE TITLE
Mup version confusion

### ISSUE THREAD

**Issue by pors** [arunoda/meteor-up-legacy#916]
Now that I have troubles deploying my meteor 1.3 apps (see #915) I started looking into and alternative version of mup, it seems we have:
1. https://github.com/arunoda/meteor-up
2. https://github.com/arunoda/meteor-up/tree/mupx
3. https://github.com/kadirahq/meteor-up

I assume that doing `npm install mup` installs the first version?

Which version is recommended for Meteor 1.3? No production quality needed yet.

Thanks
Mark

**Comment by pors** [comment]
PS Thanks for creating mup!!!! I couldn't live without it :)

**Comment by MasterJames** [comment]
Moving forward the new kadirahq version is your best bet.

**Comment by gsabran** [comment]
It would be nice to update the doc:
- on https://github.com/arunoda/meteor-up the doc says: "Mupx is the stable version. "
- on https://github.com/arunoda/meteor-up/tree/mupx the doc says: "Once mupx becomes stable we'll merge into master and release as mup."
- on https://github.com/arunoda/meteor-up/tree/mupx the doc says: "Please check arunoda/meteor-up for the stable version."

So: which one is the best stable version? How stable/unstable is the kadirahq version?

**Comment by MasterJames** [comment]
@madushan1000 I think I noticed it was you that was able to change the mup readme but I agree that the dev branch mupx needs a quick rewording too please and thank you.

I would extend that to maybe mention an expected date for a more official depreciation of these previous versions here. I imagine it will take about a month after Meteor 1.3 is officially released.

**Comment by madushan1000** [comment]
The current stable version is Mupx (https://github.com/arunoda/meteor-up/tree/mupx).
The current development version is (https://github.com/kadirahq/meteor-up).
This mup version is deprecated (https://github.com/arunoda/meteor-up). 
I'll try to add edits to clarify this issue

**Comment by gsabran** [comment]
would it make sens to replace aroduna/meteor-up's master branch by mupx ?

**Comment by madushan1000** [comment]
@arunoda what do you think?

**Comment by MasterJames** [comment]
No because it's not ready but is that possible when it is?

**Comment by MasterJames** [comment]
Sorry I meant to say not until its tested with an official Meteor 1.3 test, would be my only consideration at this point. My apologies if overstepping I'm I suppose too quick to comment but used to helping out here.
I'm also thinking about what AWS EFS really means for Apollo lately, but the nearly ready Metror 1.3 version is pretty stable and not expected to change much from the current dev branch?

**Comment by jowilhnson** [comment]
To further add to the confusion, if you install the kadira mupx version, it will cause attempts to use the original mup to no longer work, as all commands like `mup deploy` will try to run the kadira mupx, not mup and complain about not having a mup.js file. 

You can reverse this by running `npm uninstall` from within the directory you cloned kadiara mupx into and then reinstalling mup from npm. Just be aware the original mup and kadira mupx cannot coexist.
